README.md

```diff
@@ -30,6 +30,156 @@ services.config(({app, services}) => {
 services.start();
 ```
 
+### Using as CLI with Loopback 4 Architecture
+`@labshare/services` is a package for loading APIs. These APIs can be single on multiple mounted over a single thread.
+This package supports 2 modes:
+| mode  |  reason  |
+|---|---|
+| services  | LabShare basic Express app with route loading architecture |
+| lb | LabShare API LoopBack support architecture. |
+
+This cli is using `@labshare/lsc` which is a cli bundler from the LabShare project , more information [here](https://github.com/labshare/lsc)
+
+This can be defined at the .labsharerc file ( if it is not present, it will use services mode).
+
+```sh
+{
+  "mode": "lb", // or services
+  "apis":[{"name":"api"}] // array of loopback 4 apis 
+}
+
+```
+### Configuration
+
+| configuration  |  reason  |
+|---|---|
+| config.json  | services configuration file |
```

Comment:
Looks like you have a description for it, but maybe it can be more clear that you only need the "config.json" if you're running the Express app version?